### DISCUSSION TITLE
Grove VOC and eCO2 Gas Sensor (SGP30) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#199] [Q&A]
# Grove VOC and eCO2 Gas Sensor (SGP30) | Seeed Studio Wiki

Grove VOC and eCO2 Gas Sensor (SGP30)

https://wiki.seeedstudio.com/K1100-VOC-and-eCO2-Gas-Sensor-Grove-LoRa-E5/#docusaurus_skipToContent_fallback

**Comment by IcingTomato** [comment]
Mathew， shi
